chore: create script for updating RELEASE.md before PR

NeilAn99 opened the pull request:
## chore: create script for updating RELEASE.md before PR
<!--
☝️ Put your PR title up here!

"scope" could be one of our apps or packages:
- form, validator, demo, landing-page, docs...

✨ Example PR titles:
    - feat(form): implement new FormControl isValid state
    - fix(validator): correct the variable name typo causing errors
    - style(landing-page): update the logo in the landing page app
    - docs: update content project CONTRIBUTING.md
-->

Fixes #148 <!-- 👈🏻 Put the issue number here! -->

Description of changes: <!-- 👇🏻 List the changes done! -->
- Added the [prompt-sync](https://www.npmjs.com/package/prompt-sync) and [semver](https://www.npmjs.com/package/semver) packages.
- Created a script `pr-updater.js` located in the `packages` directory that can be run using `npm run release`. This script will prompt the contributor for the package changed, the type of PR, and the description(s) of changes made. Then, the script will find the corresponding RELEASE.md file and update it.

For example, the running the script:
![image](https://user-images.githubusercontent.com/54966170/206618095-687c8b71-16a9-4be8-a570-30e288281baa.png)
Will produce the following RELEASE.md:
![image](https://user-images.githubusercontent.com/54966170/206618219-07ac7f54-3010-45fd-b31c-c57665c1301e.png)

Tag a reviewer: @ayoayco 

Tasks:
- [x] I have ran the build command to make sure apps are working: `npm run build`
- [x] I have ran the tests to make sure nothing is broken: `npm run test`
- [x] I have ran the linter to make sure code is clean: `npm run lint`
- [x] I have reviewed my own code to remove changes that are not needed

<!-- THANK YOU FOR THE CONTRIBUTION! 🚀 -->

ayoayco commented on packages/pr-updater.js:
```diff
@@ -0,0 +1,94 @@
+    const formJSON = JSON.parse(fs.readFileSync(JSONfile));
+    var version;
+
+    if (type == '1') {
+        version = semver.inc(formJSON.version, 'patch');
+    }
+    else if (type == '2') {
+        version = semver.inc(formJSON.version, 'minor');
+    }
+    else {
+        version = semver.inc(formJSON.version, 'major');
+    }
```
Yeah, switch-case makes sense here, then maybe a `default` case to throw an error or loop through the question again after a message "Invalid option"

ayoayco reviewed (commented):
Thanks @NeilAn99, this is great progress for the project. We want to get this in! Can you go through a couple of suggestions first?

ayoayco commented:
Oh and one very important thing I missed: the package.json "version" property in the package needs to be updated to the same as the version that is written to the RELEASE.md. @NeilAn99

ayoayco commented on packages/pr-updater.js:
```diff
@@ -0,0 +1,94 @@
+function main() {
+    console.log("Astro Reactive PR Release Updater (ctrl-c to exit)");
+
+    console.log(`What package is changed? 
+    1. Common
+    2. Form 
+    3. Validator`);
+    const noChanged = prompt(": ");
+
+    console.log(`What is the type of PR? 
+    1. Fix
+    2. Minor Release/Feat
+    3. Major Release/Feat`);
+    const type = prompt(": ");
+
```
@Fawrish#0487 you can remove adding the version for now in your script. Just focus on the change and type. Where the type could either be "fix" or "feature"

So no need to add the big text for version but just items like:

Features
- Feature description text

Fixes
- Change description text
- dix description 2

NeilAn99 commented:
I have updated the code so that it doesn't deal with versioning, added some input checking and focused on features/fixes. Now running the script will look like:
![image](https://user-images.githubusercontent.com/54966170/206840895-9e2a292b-cc8c-4a29-8eb4-0d11ece596c4.png)
and will produce:
![image](https://user-images.githubusercontent.com/54966170/206841073-3d1cc361-d2d1-4e12-a753-f5de5a05f84a.png)
Multiple updates to the same package will look like:
![image](https://user-images.githubusercontent.com/54966170/206841090-b94599ae-7ef7-489c-9ec3-cb935c4d9973.png)
Is this acceptable? @Icyscools @ayoayco

Icyscools reviewed (approved):
LGTM! 🚀 However, please fix a conflict on a `package.json`. I think it was a line in the `publish` script. Make sure to use it from our latest main branch and it'll be safe to merge. 🦺

ayoayco commented:
Thanks @Icyscools and @NeilAn99 -- I'm merging this now! :)
